code/nodes/featureSignificance/tests/test.t

```diff
@@ -1 +1,91 @@
-1~1
+\l automl.q
+.automl.loadfile`:init.q
+
+// The following utilities are used to test that a function is returning the expected
+// error message or data, these functions will likely be provided in some form within
+// the test.q script provided as standard for the testing of q and embedPy code
+
+// @kind function
+// @category tests
+// @fileoverview Ensure that a test that is expected to fail, 
+//   does so with an appropriate message
+// @param function {(func;proj)} The function or projection to be tested
+// @param data {any} The data to be applied to the function as an individual item for
+//   unary functions or a list of variables for multivariant functions
+// @param applyType {boolean} Is the function to be applied unary(1b) or multivariant(0b)
+// @param expectedError {string} The expected error message on failure of the function
+// @return {boolean} Function errored with appropriate message (1b), function failed
+//   inappropriately or passed (0b)
+failingTest:{[function;data;applyType;expectedError]
+  applyType:$[applyType;@;.];
+  failureFunction:{[err;ret](`TestFailing;ret;err~ret)}[expectedError;];
+  functionReturn:applyType[function;data;failureFunction];
+  $[`TestFailing~first functionReturn;last functionReturn;0b]
+  }
+
+// @kind function
+// @category tests
+// @fileoverview Ensure that a test that is expected to pass, 
+//   does so with an appropriate return
+// @param function {(func;proj)} The function or projection to be tested
+// @param data {any} The data to be applied to the function as an individual item for
+//   unary functions or a list of variables for multivariant functions
+// @param applyType {boolean} Is the function to be applied unary(1b) or multivariant(0b)
+// @param expectedReturn {string} The data expected to be returned on 
+//   execution of the function with the supplied data
+// @return {boolean} Function returned the appropriate output (1b), function failed 
+//   or executed with incorrect output (0b)
+passingTest:{[function;data;applyType;expectedReturn]
+  applyType:$[applyType;@;.];
+  functionReturn:applyType[function;data];
+  expectedReturn~functionReturn
+  }
+
+\S 10
+
+// Features and targets
+featData :([]100?1f;100?1f;asc 100?1f)
+targClass:100?0b
```

Comment:
If you change the targets to also be asc/desc the passing tests for this will be more robust allowing you to check that the correct 'significant feature' is being returned